packages/mysql/lib/mysql_p.js

```diff
@@ -254,19 +252,22 @@ function captureOperation(name) {
       }
     }
 
-    subsegment.addSqlData(createSqlData(config, command));
+    subsegment.addSqlData(createSqlData(config, args.values, args.sql));
     subsegment.namespace = 'remote';
 
     return command;
   };
 }
 
-function createSqlData(config, command) {
-  var commandType = command.values ? PREPARED : null;
-
+function createSqlData(config, values, sql) {
```

Comment:
Hi, thanks for this contribution! 
You mentioned this issue + the fix for it in the bug report, maybe keeping `createSqlData(config, args)` with 2 parameters might be better to remain consistent with the [same function in postgres](https://github.com/aws/aws-xray-sdk-node/blob/master/packages/postgres/lib/postgres_p.js#L125)?

Reply:
Well... other than the number of arguments, the calls to createSqlData are not very consistent in the current state, primarily because the value getting passed in to createSqlData in mysql_p can either be a command or a _promise_ to return results (which is the whole reason this PR exists in the first place).

As far as the number of arguments to createSqlData in mysql_p.js, I could pass in the args which consistently contains both the SQL and parameters, which would keep it at two arguments; albeit the second argument by necessity is different than what gets passed in to createSqlData in postgres_p.js.  In untyped languages I always found it easier to explicitly accept discrete values as opposed to a composited object, but it's a nit.  If two arguments is important, I can make that work and resubmit so I can log SQL with MySQL in XRay.

Reply:
Yea they definitely do different things, this was just a nit pick to target as much consistency as possible. There isn't a strong preference for either option on my end, so I'll leave that decision up to you. I think a small comment explaining how/why this function is different can also be very helpful for anyone looking back on this implementation

Reply:
Ok, added function notes per your request